integration/without_composer_json_test.go

```diff
@@ -0,0 +1,70 @@
+package integration_test
```

Comment:
I think if we change the detection criteria of this buildpack (see comment on README) then this test can go. For future reference, I would expect this to be something tested on the unit test level  in the detect tests instead of the integration test level.

Reply:
it's gone